rpcserver.go

```diff
@@ -6371,21 +6371,6 @@ func (r *rpcServer) GetNetworkInfo(ctx context.Context,
 func (r *rpcServer) StopDaemon(_ context.Context,
 	_ *lnrpc.StopRequest) (*lnrpc.StopResponse, error) {
 
-	// Before we even consider a shutdown, are we currently in recovery
```

Comment:
Is it a good idea to disable this check? The comment mentions that without this, the rescan wouldn't continue unless explictly started that way. Differing behaviour needs to be documented (@losh11).

Reply:
It says you just need to remember to unlock the wallet next time with the recovery window option, which we do do. So its not an issue to allow a shutdown.